src/Microsoft.Diagnostics.ExtensionCommands/NativeAddressHelper.cs

```diff
@@ -68,25 +73,144 @@ internal IEnumerable<DescribedRegion> EnumerateAddressSpace(bool tagClrMemoryRan
                     {
                         foreach (ClrMemoryPointer mem in ClrMemoryPointer.EnumerateClrMemoryAddresses(clrRuntime))
                         {
-                            var found = ranges.Where(m => m.Start <= mem.Address && mem.Address < m.End).ToArray();
+                            var found = rangeList.Select((DescribedRegion Region, int Index) => (Region, Index)).Where(m => m.Region.Start <= mem.Address && mem.Address < m.Region.End).ToArray();
 
-                            if (found.Length == 0)
+                            if (found.Length == 0 && mem.Kind != ClrMemoryKind.GCHeapReserve)
+                            {
                                 Trace.WriteLine($"Warning:  Could not find a memory range for {mem.Address:x} - {mem.Kind}.");
```

Comment:
Do we still need this Trace since it is printed to console below? `!logging enable` defaults to the windbg console.

Reply:
It only prints the first one.  I've been using Trace to spew some diagnostics when I encounter a weird dump.  I'd like to leave it, if that's ok.  At least until I throw it against a bigger/weirder set of dumps.

Reply:
No problem. It fine with me.